firmware/Core/Src/telecommands/eps_telecommands.c

```diff
@@ -0,0 +1,24 @@
+#include <eps_drivers/eps_commands.h>
+#include <eps_drivers/eps_types.h>
+
+#include <telecommands/eps_telecommands.h>
+
+#include <stdio.h>
+
+/// @brief Triggers the EPS watchdog. No args.
+/// @return 0 on success, 1 on failure.
+uint8_t TCMDEXEC_eps_watchdog(const uint8_t *args_str, TCMD_TelecommandChannel_enum_t tcmd_channel,
```

Comment:
Suggest TCMDEXEC_EPS_watchdog (namespace case)

Reply:
I think we'll keep all command names fully lowercase to avoid errors while planning/uplinking telecommands. The `xxx` part in `TCMDEXEC_xxx` should match the telecommand's name string (defined in the telecommand table), and thus remains lowercase. 

Ensuring that the subsystem is explicitly stated at the start of the telecommand name is 100% a requirement though, regardless of capitalization.

Reply:
Got it. All good re namespaces.